web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html

```diff
@@ -53,8 +53,14 @@
                     class="text-muted gn-status"
                     data-ng-class="{'text-success': md.mdStatus == 2, 'text-warning': md.mdStatus == 4}"
                     data-ng-if="md.mdStatus < 50 && isMdWorkflowEnable"
-                    >{{::md.statusWorkflow | getStatusLabel}}</small
                   >
+                    <span data-ng-if="md.draft = 'e'"
```

Comment:
```suggestion
                    <span data-ng-if="md.draft !== 'y'"
```
this could fix the 'show only one status per line' issue, but not sure what the implication is on other setups @fxprunayre